bridge_adaptivity/bridge_lti/outcomes.py

```diff
@@ -0,0 +1,204 @@
+"""
+Based on OpenEdx lti_provider application.
+Helper functions for managing interactions with the LTI outcomes service defined in LTI v1.1.
+"""
+
+import logging
+import uuid
+
+import requests
+import requests_oauthlib
+from lxml import etree
+from lxml.builder import ElementMaker
+from requests.exceptions import RequestException
+
+from .models import OutcomeService
+
+log = logging.getLogger(__name__)
+
+
+def store_outcome_parameters(request_params, sequence, lti_consumer):
+    """
+    Determine whether a set of LTI launch parameters contains information about
+    an expected score, and if so create a GradedAssignment record. Create a new
+    OutcomeService record if none exists for the tool consumer, and update any
+    incomplete record with additional data if it is available.
+    """
+    result_id = request_params.get('lis_result_sourcedid', None)
+
+    """
```

Comment:
I would use """ for doc strings not for the inline comments why don't just use `#` character?